docs/fundamentals/networking/httpclient.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Make HTTP requests with the HttpClient
+description: Learn how to make HTTP requests and handle responses with the HttpClient in .NET.
+author: IEvangelist
+ms.author: dapine
+ms.date: 08/10/2022
+---
+
+# Make HTTP requests with the HttpClient
+
+Hypertext Transfer Protocol (or HTTP) is a protocol for requesting resources from a web server. The <xref:System.Net.Http.HttpClient?displayProperty=fullName> class exposes the ability to send HTTP requests and receive HTTP responses from a resource identified by a URI. There are many types of resources that are available on the web, and HTTP defines a set of request methods for accessing these resources. The request methods are differentiated via several factors, first by their _verb_ but also by the following characteristics:
+
+- A request method is **_idempotent_** if it can be successfully processed multiple times without changing the result. For more information, see [RFC 7231: Section 4.2.2 Idempotent Methods](https://datatracker.ietf.org/doc/html/rfc7231#section-4.2.2).
```

Comment:
This set of topics and the table is important, but it doesn't seem important to lead with. Feel like an advanced set of topics where we should lead with "how do I get HTTP GET to work?"

Reply:
I can see that, but this isn't really advanced - in my opinion, it's more conceptual and like an overview. Perhaps, we should have a dedicated overview article that would then lead into this article, does that make sense? It could include diagrams to help visualize the request/response flow that the HTTP protocol imposes. Thoughts?